eno-core/src/main/java/fr/insee/eno/core/processing/impl/DDIInsertDeclarations.java

```diff
@@ -22,28 +22,28 @@ public void apply(EnoQuestionnaire enoQuestionnaire) { // Note: code is a bit cl
         //
         for (Sequence sequence : enoQuestionnaire.getSequences()) {
```

Comment:
cognitive complexity ?

Reply:
True, there is actually a lack of testing on this class also, this will be the purpose of a future pr